aiida/utils/calculation.py

```diff
@@ -39,3 +39,7 @@ def add_source_info(node, func):
         node._set_attr("source_file", source)
     except (IOError, OSError):
         pass
+    try:
+        node._set_attr("namespace", func.func_globals["__name__"])
+    except:
```

Comment:
It's not good to have a generic 'except' (this e.g. will catch also a CTRL+C...).
Do you know who might rise an exception? can func.func_globals do it? 
In the "worst" case, at least do `except Exception`

Reply:
Actually, I don't know what exception might be risen here. The idea behind generic 'except' was a general fallback to silently skip setting this attribute should anything unexpected happen. I can replace it with ``except Exception``.